src/Core/AndroidNative/maui/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,3 @@
+<resources>
+    <string name="maui_empty_unused"></string>
```

Comment:
Yeah, if we have to put one, maybe it's worth putting `library_name=.NET MAUI`? I feel like I remember some convention for that.

Reply:
And a comment, saying we need at least 1 resource.

Reply:
The problem with using `library_name` is that it might be used on other libraries. And depending on the order of import you might override something since the resources will overlay on top of each other. 

I would prefer this be unique if possible so we don't trip over any existing resources.